include/kernel.h

```diff
@@ -4223,7 +4223,9 @@ struct k_mem_slab {
 	char *buffer;
 	char *free_list;
 	uint32_t num_used;
-
+#ifdef CONFIG_MEM_SLAB_TRACE_MAX_UTILIZATION
+	uint32_t max_used;
+#endif
 	_OBJECT_TRACING_NEXT_PTR(k_mem_slab)
 	_OBJECT_TRACING_LINKED_FLAG
```

Comment:
It seems that this empty line before `_OBJECT_TRACING` stuff was intentional. I think it should be kept.

Reply:
Recovered empty line.